Sources/Controllers/AnimatableModalViewController.swift

```diff
@@ -179,4 +179,10 @@ open class AnimatableModalViewController: UIViewController, PresentationDesignab
     super.viewDidAppear(animated)
     configureDismissalTransition()
   }
+
+  open override func viewDidLayoutSubviews() {
```

Comment:
I think that's safe to use `viewDidLayoutSubviews`, it will update the UI when any UI has been changed. Another way would be `viewWillTransitionToSize:withTransitionCoordinator:` for size class. But I think `viewDidLayoutSubviews` is fine.